pkg/constants/common.go

```diff
@@ -16,13 +16,15 @@ limitations under the License.
 
 package constants
 
-type ResultMessage struct {
-
-	Data	interface{} `json:"data"`
+type MessageResponse struct {
+	Message interface{} `json:"message"`
```

Comment:
Is the difference between ResultMessage and MessageResponse just the key of response data?